dydx-testnet-4/gentx/gentx-RockawayX.json

```diff
@@ -0,0 +1 @@
+{"body":{"messages":[{"@type":"/cosmos.staking.v1beta1.MsgCreateValidator","description":{"moniker":"RockawayX","identity":"EB935356000A9562","website":"https://rockawayx.com/infrastructure","security_contact":"infra@rockawayx.com","details":"RockawayX Infra is an institution-grade bare-metal staking operator"},"commission":{"rate":"0.050000000000000000","max_rate":"0.200000000000000000","max_change_rate":"0.010000000000000000"},"min_self_delegation":"1","delegator_address":"dydx19c0vvpnxf4vcxnvrxw854vhmz90akw5tugtdy0","validator_address":"dydxvaloper19c0vvpnxf4vcxnvrxw854vhmz90akw5tekam07","pubkey":{"@type":"/cosmos.crypto.ed25519.PubKey","key":"gHCfOaZHHgRSIlW36q9Kah6QhpbFbsoSz457u8QGiF0="},"value":{"denom":"dv4tnt","amount":"50000000000"}}],"memo":"47cb90109f169ac4a18154b16517d38c19830773@10.36.213.100:26656","timeout_height":"0","extension_options":[],"non_critical_extension_options":[]},"auth_info":{"signer_infos":[{"public_key":{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"A7qsusFbfP/CsmuyG5Rc/cS15t2ZyoWblv8JdIDJ6vNq"},"mode_info":{"single":{"mode":"SIGN_MODE_DIRECT"}},"sequence":"0"}],"fee":{"amount":[],"gas_limit":"200000","payer":"","granter":""},"tip":null},"signatures":["iXMXTIHsLgOJcLDJDBaZfAVUc6PB3fYsMtF0drfHf64T6yM5gN6EcxqCvzHrEFeK/nqT1jR1xrJbty3GvgtTiA=="]}
```

Comment:
Hi, @Tomas-Eminger 
The [Prepare for Genesis](https://v4-teacher.vercel.app/validators/preparing_for_genesis) guide is a little different this time, the test token is `adv4tnt` and the self-delegation amount is `500000000000000000000000adv4tnt`